internal/learn/iota-token/staking-iota.md

```diff
@@ -0,0 +1,99 @@
+
+You can read how to stake and calculate the rewards you can earn [here](https://shimmer.network/claim).
+- With the launch of the Shimmer network, these generated Shimmer tokens will be distributed to the IOTA stakers. This distribution will be implemented as a functionality within the Firefly wallet. As long as the Shimmer network is not yet launched, Firefly will hold the record of rewards for the IOTA token holder. 
+
+## 2.) Staking IOTA tokens to secure the Assembly network
+
+Assembly, the Smart contract implementation connecting and governing thousands of independent Smart contract blockchains on top of IOTA's Tangle will have a deep connection with the IOTA tangle. The security of the Assembly network relies on the underlying security of the IOTA tangle as it solves the interconnection and exchange layer between different chains in Assembly and provides a secure anchor and immutable audit trail for state updates of IOTA smart contracts chains in Assembly.
+
+Also, the Tangle, with its newly implemented tokenization abilities and multi-asset functionalities, will be the home of all kinds of interoperable tokens, able to be freely exchanged between accounts on smart contract chains and the Tangle.
+
+In Assembly [token launch and fair distribution sheme](https://blog.assembly.sc/assembly-fair-launch-token-distribution/) it is planned to distribute a total of 20% of the Assembly supply to IOTA token holders. 
+The first distribution phase happens in the 90 days as described above.
+To further incentivize IOTA token owners to secure the IOTA network and help Assembly in its start phase, another ASMB distribution will happen during the first 21 months of the newly launched Assembly network.
+
+![assembly supply](/img/learn/iota-token/assembly-supply.png)
+
+- the amount of ASMB tokens that can be distributed during these 21 months depends totally on the number of tokens that are still undistributed after the initial 90 days of staking. 
+- If less than 100% of IOTA tokens have been staked for Assembly (which would distribute 8 Billion ASMB tokens to IOTA stakers), the remaining ASMB tokens will be distributed together with the planned 12 Billion ASMB tokens to IOTA stakers.
+If only 6 Billion ASMB tokens could be distributed in the first 90 days, 14 Billion ASMB tokens will be available during the second 21 month staking period.
+- the ASMB distribution during the second staking period will happen on a **daily fixed rate**
+- assuming 14 Billion ASMB tokens will be available to distribute, this would lead to a daily payout of 
+    - 21.909.233 ASMB tokens to the IOTA stakers over 639 days
+    - the amount of individual received ASMB tokens per day depends on the total percentual part an IOTA staker holds of the total daily staked amount of IOTA tokens.
+    - the fewer total IOTA tokens are staked, the higher the individual payout of theis 21.909.233 ASMB tokens per day for an IOTA staker.
```

Comment:
```suggestion
- The ASMB token distribution during this staking period will happen on a **daily fixed rate**:
    - Assuming 14 billion ASMB tokens will be available to distribute, this would lead to a daily payout of 21.909.233 ASMB tokens to the IOTA stakers over 639 days.
    - The amount of individually received ASMB tokens per day depends on how much IOTA tokens the holder staked compared to the total amount of IOTA tokens staked on that day.
    - The fewer total IOTA tokens staked, the higher the individual payout per day for an IOTA staker.
```